### PR TITLE
Read the name and properties of the instance only in `getInstance` service.

### PR DESCRIPTION
**Pull request by xzyJavaX**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #[10421](https://github.com/apache/skywalking/issues/10421)
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
If you want to add this, please make sure query dao would use this.

File: docs/en/changes/changes.md [hunk 1]
```diff
@@ -104,6 +104,7 @@
 * `Scope` in the Entity of Metrics query v1 protocol is not required and automatical correction. The scope is determined based on the metric itself.
 * Add explicit `ReadTimeout` for ConsulConfigurationWatcher to avoid `IllegalArgumentException: Cache watchInterval=10sec >= networkClientReadTimeout=10000ms`.
 * Fix `DurationUtils.getDurationPoints` exceed, when `startTimeBucket` equals `endTimeBucket`.
+* Storage-elasticsearch-plugin supports query of selected fields.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:107]:
```suggestion
* Read the name and properties of the instance only in `getInstance` service.
```

**Comment by wu-sheng** [comment]
@kezhenxu94 Please take a look.

**Review by kezhenxu94 — APPROVED** [review]
Code level changes look good to me, but this may bring some troubles when future contributors use the `getInstance` and need the other fields like instance uuid, or id

**Comment by wu-sheng** [comment]
> Code level changes look good to me, but this may bring some troubles when future contributors use the `getInstance` and need the other fields like instance uuid, or id, they might have no idea why these two fields are null/empty

@xzyJavaX You are better to keep consistent logic in JDBCMetadataQueryDAO`

**Review by wu-sheng — CHANGES_REQUESTED** [review]
I think it is wrong to build only two columns, even for BanyanDB.

@lujiajing1126 It seems BanyanDB implementation doesn't return required fields.

**Comment by wu-sheng** [comment]
Close for now until we have a real use case to use Sources of ElasticSearch accessing.

**Pull request closed without merging**